cognitedata/reveal
# fix: skip pointcloud tiles loading if numPoints is zero

#### Type of change
![Bug](https://img.shields.io/badge/Type-Bug-red) 

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-776

## Description :pencil:
When Reveal attempts to download the files for sectors with 0 points, the .bin files are non-existent than loading sector will be skipped which will not cause any error.

## How has this been tested? :mag:
In `examples` with `modelId = 3421113048655780` with `revisionId = 762530969131593` from `bluefield` cluster


## Test instructions :information_source:
1. Load https://localhost:3000/migration?project=3d-test-az&env=bluefield-3d&modelId=3421113048655780&revisionId=762530969131593
2. Move the camera around the scene and check the developer tool console log for error


## Checklist :ballot_box_with_check:
